js-jasmine/spec/Shop.spec.js

```diff
@@ -17,7 +17,7 @@ describe("Gilded Rose", function() {
 
 });
 
-describe('Class: Shop()', function() {
+describe('Shop()', function() {
```

Comment:
- I like how you have separated the tests for each type of item available for Gilded Rose.
- Tests are clear and easy to follow!